### ISSUE TITLE
Windows updates wants to revert my WSLg install from 1.0.23 to 1.0.22

### ISSUE THREAD

**Issue by quantummixer** [microsoft/wslg#302]
# Environment

```none
Windows build number: 10.0.21390.1010
Your Distribution version: Ubuntu Release: 18.04
Your WSLg version: Window Subsystem for Linux WSLg Preview version 1.0.23
```

# Steps to reproduce

As a solution to my previous issue #288 , I had to manually install WSLg MSI package v1.0.23 to fix the multi-display monitor issue. 

Now `Windows Update` wants to downgrade my WSLg back to v1.0.22.  

**WSL logs**: 

Not relevant here.

#  Expected behavior

`Windows Update` should not down rev my WSLg version from 1.0.23 back down to 1.0.22.

# Actual behavior

A screen shot of the `Windows Update` is provided for reference.
![windows_update_trying_roll_back_wslg_version](https://user-images.githubusercontent.com/71977913/122288030-77848a80-cea6-11eb-8c81-0c4243318607.PNG)

**Comment by bpulliam** [comment]
The fix for this issue is in 1.0.24.  Starting with that release, you should not be offered older release builds if a prerelease is installed.

**Comment by quantummixer** [comment]
Thank you and your team for the awesome support!  
I will stop my Windows Update for now, until 1.0.24 becomes available.
